examples/vmi-masquerade.yaml

```diff
@@ -40,10 +40,18 @@ spec:
       userData: |-
         #!/bin/bash
         echo "fedora" |passwd fedora --stdin
-        ip -6 addr add fd10:0:2::2/120 dev eth0
-        sleep 5
-        ip -6 route add default via fd10:0:2::1 src fd10:0:2::2
-        yum install -y nginx
-        systemctl enable nginx
-        systemctl start nginx
+      networkData: |
+        version: 2
+        ethernets:
+          eth0:
+            addresses:
+              - fd10:0:2::2/120
```

Comment:
This will make it solely IPv6, but it was dual before.
Could you please check if you can define IPv4 auto and IPv6 static at the same time?

Reply:
Good catch.

Adding the `dhcp4: true` does it; added it also to the `GetIPv6NetworkData` function (thus ensuring the old behavior is preserved).